ci-operator/step-registry/openshift-qe/kube-burner-smoke/openshift-qe-kube-burner-smoke-commands.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+python --version
+pushd /tmp
+python -m virtualenv ./venv_qe
+source ./venv_qe/bin/activate
+
+ES_PASSWORD=$(cat "/secret/password")
+ES_USERNAME=$(cat "/secret/username")
+export ES_SERVER="https://$ES_USERNAME:$ES_PASSWORD@search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com"
+git clone https://github.com/cloud-bulldozer/e2e-benchmarking --depth=1
+pushd e2e-benchmarking/workloads/kube-burner-ocp-wrapper
+
+current_worker_count=$(oc get nodes --no-headers -l node-role.kubernetes.io/worker=,node-role.kubernetes.io/infra!=,node-role.kubernetes.io/workload!= --output jsonpath="{.items[?(@.status.conditions[-1].type=='Ready')].status.conditions[-1].type}" | wc -w | xargs)
+set_pods_per_node(){
+    current_pod_count=$(oc get nodes --selector="node-role.kubernetes.io/worker=,node-role.kubernetes.io/infra!=,node-role.kubernetes.io/workload!=" -o name | cut -d '/' -f 2 | xargs -I {} oc get pods --field-selector=status.phase=Running,spec.nodeName={} -o name -A | wc -l)
```

Comment:
I dont think this type of labeling exists on rosa clusters, do we want to verify this works for all set ups?

Reply:
All our kube-burner runs use [this](https://github.com/cloud-bulldozer/go-commons/blob/main/ocp-metadata/types.go#L25) same labels to calculate the worker nodes. Since it is working for every run so far, it is expected to work here as well.